inductiva/simulation/simulator.py

```diff
@@ -81,5 +87,28 @@ def run_async(
         return api.run_async_simulation(
             self.api_method_name,
             input_dir,
-            params=kwargs,
+            params=[kwargs],
+        )
+
+    def run_pipeline(self,
+                     input_dir: types.Path,
+                     *_args,
+                     output_dir: Optional[types.Path] = None,
+                     pipeline: List[Command],
+                     track_logs: bool = False) -> pathlib.Path:
+        """Run a pipeline of commands.
+        Args:
+            working_dir: Path to the directory where the pipeline will be 
+            executed.
+            pipeline: List of commands to be executed in the simulation API.
```

Comment:
I did like this instead @IvanPombo, in response to your comment above

Reply:
I did like this instead @IvanPombo, in response to your comment above

Reply:
I did not get it... `working_dir` is just on the comment, not on the actual arguments, that is why I was confused.